Controllers/Website/Administration/NewNpc.php

```
@@ -0,0 +1,71 @@
+    {
+        switch ($_SERVER['REQUEST_METHOD']) {
+            case 'GET':
+                return $this->get(array());
+            case 'POST':
+                return $this->post(array());
+	        default:
+	        	return 405;
+        }
+    }
+
+    public function get(array $args): int
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)